charts/tfy-gpu-operator/values.yaml

```diff
@@ -117,7 +117,14 @@ aws-eks-gpu-operator:
       resources:
         requests:
           cpu: 10m
-          memory: 300Mi
+          memory: 100Mi
+      tolerations:
+        ## @skip aws-eks-gpu-operator.node-feature-discovery.gc.tolerations[0]
+        - key: nvidia.com/gpu
+          operator: Exists
+          effect: PreferNoSchedule
```

Comment:
I am a little concerned if GC gets put on a GPU node and it might cause Karpenter to not drop the GPU node when consolidation is turned off.
Should I force gc to be on non-GPU nodes via a NotIn instance size affinity?

Reply:
> Should I force gc to be on non-GPU nodes via a NotIn instance size affinity?

I think this is a safer option without loosing much.